windows/UWP/ReactTestApp.vcxproj

```diff
@@ -24,6 +24,7 @@
     <ReactAppUniversalDir Condition="'$(ReactAppUniversalDir)'==''">$(ReactAppWinDir)\UWP</ReactAppUniversalDir>
     <ReactAppGeneratedDir Condition="'$(ReactAppGeneratedDir)'==''">$(MSBuildProjectDirectory)\..\..</ReactAppGeneratedDir>
     <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)'==''">$([MSBuild]::GetDirectoryNameOfFileAbove($(SolutionDir), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
+    <ReactNativeDir Condition="'$(ReactNativeDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(SolutionDir), 'node_modules\react-native\package.json'))\node_modules\react-native\</ReactNativeDir>
```

Comment:
This looks correct. In RNW, we have some hoisting assumptions:
https://github.com/microsoft/react-native-windows/blob/bd6a818ab994d3e6f0a729d41f8165f062292d8a/vnext/Microsoft.ReactNative.Cxx/Microsoft.ReactNative.Cxx.vcxitems#L7-L8

Mind making this ready for review? Or are you looking into other issues?

Reply:
Made ready - not sure about the failing test. Do I just need to modify the message to follow the convention specified?

Reply:
It refers to the commit message: https://github.com/microsoft/react-native-test-app/blob/trunk/CONTRIBUTING.md#commit-messages

But it's fine. It's not a required check and we can modify it before merging.